docs/metrics/metrics-queries/metric-query-error-messages.md

```diff
@@ -5,7 +5,24 @@ sidebar_label: Error Messages
 description: Sumo issues errors when metric queries run too long or match too many time series.
 ---
 
-This page describes error messages that are presented for long-running metric queries and metric queries that return too many results.
+This page describes warning and error messages that are presented for long-running metric queries and metric queries that return too many results.
+
+# Contents
```

Comment:
Please delete the table of contents you added. A page TOC is auto-generated from page headings, displayed on the right side of each help page.